backend/app/api/src/helpers/SetupStepsUpdater.ts

```diff
@@ -0,0 +1,194 @@
+import {
+    Organization,
+    OrganizationRegistrationPeriod,
+    Platform,
+} from "@stamhoofd/models";
+import { SetupStepType, SetupSteps } from "@stamhoofd/structures";
+
+export class SetupStepUpdater {
+    private static readonly STEP_TYPE_OPERATIONS: Record<SetupStepType, (setupSteps: SetupSteps,
+        organization: Organization,
+        platform: Platform) => void> = {
+        [SetupStepType.Groups]: this.updateStepGroups,
+        [SetupStepType.Premises]: this.updateStepPremises
+    }
+
+    static async updateSetupStepsForAllOrganizationsInCurrentPeriod({
+        batchSize, stepTypes, platforms
+    }: { batchSize?: number, stepTypes?: SetupStepType[], platforms?: Platform[] } = {}) {
+
+        if(!platforms) {
+            platforms = await Platform.all();
+        }
+
+        for (const platform of platforms) {
```

Comment:
Zou dit wrappen in een QueueHandler.schedule zodat er altijd maximaal één `updateSetupStepsForAllOrganizationsInCurrentPeriod` kan runnen, en bv. geen 10. Via QueueHandler.cancel(queuenaam) kan je net voor je gaat updaten, de lopende queue clearen zodat er altijd maximaal één in wacht staat, en je dus niet 10 keer na elkaar gaat updaten